test/integration/etcd2_client_test.lua

```diff
@@ -1,5 +1,5 @@
 local t = require('luatest')
-local g = t.group()
+local g = t.group('integration.etcd2_client_test')
```

Comment:
why? the default value is derived from filename, in this case - "integration.etcd2_client" (without "_test")

Reply:
For debug purposes only. Forgot to remove it